src/openlcb/TractionTrain.cxx

```diff
@@ -62,6 +62,12 @@ TrainNodeForProxy::TrainNodeForProxy(TrainService *service, TrainImpl *train)
     service_->register_train(this);
 }
 
+TrainNodeForProxy::~TrainNodeForProxy()
+{
+    // breaks unit tests due to bugs
```

Comment:
Perhaps an @todo comment to come back and fix this?

Reply:
done